### ISSUE TITLE
react-json-tree: clientside exception with nextjs 13

### ISSUE THREAD

**Issue by dlcoffee** [reduxjs/redux-devtools#1279]
hi! i'm currently using `react-json-tree` for a project without issues. i tried to upgrade my project from nextjs 12 -> 13 but it seems like something is breaking when running in production mode. i've set up an example repo here with steps to reproduce: https://github.com/dlcoffee/json-viewer-bug

i can't tell if this is an issue with `react-json-tree` or if it's something to do with the way nextjs 13 is working. for the record, the app loads fine in development (`npm run dev`).

<img width="1800" alt="Screenshot 2022-11-17 at 17 59 22" src="https://user-images.githubusercontent.com/2092822/202599723-123fc1be-ae0d-4296-abef-8b1b5b4b43c0.png">

**Comment by ciruz** [comment]
Am I missing something, where is Redux in your example project or in your error messages involved?

**Comment by dlcoffee** [comment]
> Am I missing something, where is Redux in your example project or in your error messages involved?

redux isn't relevant in my example. i opened up the issue in this repo because the `react-json-tree` package is in it: https://github.com/reduxjs/redux-devtools/tree/main/packages/react-json-tree

**Comment by AlexSapoznikov** [comment]
I am having the same issue.

**Comment by dlcoffee** [comment]
> I am having the same issue. Did you manage to figure it out @dlcoffee ?

no, I ended up to switching to a different json viewer tool, since it wasn't critical to use this package.

**Comment by znatty22** [comment]
@dlcoffee Which json viewer tool did you switch to? I'm trying to use `react-json-view` in a nextjs 13 app and I'm having trouble getting it to work
